sdks/python/apache_beam/utils/annotations_test.py

```diff
@@ -30,6 +30,7 @@ class AnnotationTests(unittest.TestCase):
   # Note: use different names for each of the the functions decorated
   # so that a warning is produced for each of them.
   def test_deprecated_with_since_current_message(self):
+    warnings.resetwarnings()
```

Comment:
Move to setUp() ? (https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp)

Reply:
ok let me test that

Reply:
Done